### ISSUE TITLE
App does not run

### ISSUE THREAD

**Issue by DevCubeHD** [electron-userland/electron-packager#642]
I simply need some help because after packaging my app it does not start.
No window comes up, it simply does nothing.

**My package.json:**
`{
  "name": "CloudSystem.iO",
  "productName": "CloudSystem.iO",
  "description": "A minimal Electron application",
  "version": "1.0.0",
  "main": "app.js",
  "private": true,
  "scripts": {
    "start": "electron .",
    "package": "electron-packager . CloudSystem.iO --platform=win32 --arch=x64 --overwrite --out ./dist/build --icon ./build/icon.ico",
    "build": "build --win --x64 --prepackaged ./dist/build/CloudSystem.iO-win32-x64"
  },
  "author": "Felix Gaebler",
  "devDependencies": {
    "electron": "1.4.15",
    "electron-cookies": "1.1.0",
    "extract-zip": "1.6.0"
  },
  "dependencies": {}
}
`
I hope you can help me a bit :)
When I run the build without a path it creates the package in dist/win-unpacked. This binary is execuable! But some scripts are not working and the logo is missing on the platform

**Comment by tomasantunes** [comment]
I have the same problem.
I ran this command and the executable doesn't launch.

```
electron-packager . AppName --overwrite --platform=win32 --arch=x64 --icon=./img/logo.ico
```

**Comment by malept** [comment]
Thanks for filing an issue! In order to help you with your problem, we're going to need more information about it. In particular:

* Console output when you run electron-packager with the environment variable `DEBUG=electron-packager`.

* Please provide either a failing minimal testcase (with code) or detailed steps to reproduce your
problem. Using [electron-quick-start](https://github.com/electron/electron-quick-start) is a good starting point.

**Comment by ChannelOne** [comment]
Same issue, I am building my project [MDE](https://github.com/ChannelOne/MDE/tree/release-v0.1.2) with branch `release-v0.1.2` on Windows 10 Creators Update

Building output:
```
PS C:\Users\cdzos\Documents\MyProjects> electron-packager.cmd .\MDE MDE --overwrite --platform=win32 --arch=x64 --icon=./MDE/build_resources/icon.ico
  electron-packager Electron Packager 8.6.0 +0ms
  electron-packager Node v7.9.0 +3ms
  electron-packager Host Operating system: win32 (x64) +2ms
  electron-packager Packager Options: {"_":[".\\MDE","MDE"],"all":false,"deref-symlinks":true,"download":{"strictSSL":true},"overwrite":true,"prune":true,"quiet":false,"platform":"
"x64","icon":"./MDE/build_resources/icon.ico","dir":".\\MDE","name":"MDE","protocols":[]} +0ms
  electron-packager Target Platforms: win32 +1ms
  electron-packager Target Architectures: x64 +1ms
  electron-packager Inferring appVersion from version in C:\Users\cdzos\Documents\MyProjects\MDE\package.json +19ms
  electron-packager Inferring target Electron version from electron in C:\Users\cdzos\Documents\MyProjects\MDE\package.json +11ms
  electron-packager Application name: MDE +2ms
  electron-packager Target Electron version: 1.6.6 +3ms
  electron-packager Ignored path regular expressions: [ '/node_modules/electron($|/)',
  '/node_modules/electron-prebuilt(-compile)?($|/)',
  '/node_modules/electron-packager($|/)',
  '/\\.git($|/)',
  '/node_modules/\\.bin($|/)',
  '\\.o(bj)?$' ] +1ms
  electron-packager Downloading Electron with options {"strictSSL":true,"platform":"win32","arch":"x64","version":"1.6.6"} +11ms
Packaging app for platform win32 x64 using electron v1.6.6
  electron-packager Creating C:\Users\cdzos\AppData\Local\Temp\electron-packager\win32-x64-template +1s
  electron-packager Extracting C:\Users\cdzos\.electron\electron-v1.6.6-win32-x64.zip to C:\Users\cdzos\AppData\Local\Temp\electron-packager\win32-x64-template +8ms
  electron-packager Initializing app in C:\Users\cdzos\AppData\Local\Temp\electron-packager\win32-x64\MDE-win32-x64 from C:\Users\cdzos\AppData\Local\Temp\electron-packager\win32-x
mplate +4s
  electron-packager Ignored paths based on the out param: [ 'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-darwin-ia32',
  'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-darwin-x64',
  'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-darwin-armv7l',
  'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-linux-ia32',
  'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-linux-x64',
  'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-linux-armv7l',
  'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-mas-ia32',
  'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-mas-x64',
  'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-mas-armv7l',
  'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-win32-ia32',
  'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-win32-x64',
  'C:\\Users\\cdzos\\Documents\\MyProjects\\MDE-win32-armv7l' ] +29ms
  electron-packager Running npm prune --production +54s
  electron-packager Renaming electron.exe to MDE.exe in C:\Users\cdzos\AppData\Local\Temp\electron-packager\win32-x64\MDE-win32-x64 +17s
  electron-packager Running rcedit with the options {"version-string":{},"file-version":"0.1.2","product-version":"0.1.2","icon":"./MDE/build_resources/icon.ico"} +7ms
  electron-packager Moving C:\Users\cdzos\AppData\Local\Temp\electron-packager\win32-x64\MDE-win32-x64 to C:\Users\cdzos\Documents\MyProjects\MDE-win32-x64 +976ms
```

But when I run the `exe`, no window appears, nothing show in the Terminal, I don't know what happens.

**Comment by malept** [comment]
As I mentioned in my previous comment, it would be helpful if someone provided a minimal testcase, instead of a full app, which produces the same result. That way it's easier to pinpoint what's causing the problem.

**Comment by tomasantunes** [comment]
I just tested with the electron-quick-start app and the result is the same.

**Comment by malept** [comment]
I'm thinking this has something to do with https://github.com/electron/electron/issues/9323, because this hasn't been a problem before yesterday.

**Comment by malept** [comment]
Yep, I just released `extract-zip` 1.6.5, and I can no longer reproduce this.
